### PR TITLE
ZCS-1145: Universal UI - Fit and finish on appointment right sidebar suggest time interface

### PR DESCRIPTION
**Pull request by kalpeshhpatel**
Summary of Changes:
* Updated look and feel of new calendar appointment suggest time right sidebar

Changes:
* Added updated required CSS and rules in `skin.css`, `zm.css` and `skin.properties`
* Added updated required icons

https://jira.corp.synacor.com/browse/ZCS-1145

### CODE REVIEW

File: WebRoot/js/zimbraMail/calendar/view/ZmLocationSuggestionView.js [hunk 1]
```diff
@@ -105,6 +105,7 @@ function(itemDiv, ev) {
 ZmLocationSuggestionView.prototype._setNoResultsHtml =
 function() {
     var	div = document.createElement("div");
```
Comment (tarangkhandelwal) [inline WebRoot/js/zimbraMail/calendar/view/ZmLocationSuggestionView.js]: I think there's a tab between "var" & "div" , maybe we can change it to space with this change ?

Comment (kalpeshhpatel) [reply WebRoot/js/zimbraMail/calendar/view/ZmLocationSuggestionView.js]: Fixed: https://github.com/Zimbra/zm-web-client/pull/154/commits/ed2bb21d4bca8573e9059d8e9694e9b0316b417a

Thanks

File: WebRoot/skins/_base/base4/skin.css [hunk 2]
```diff
@@ -916,6 +916,40 @@ we are hiding element with that class
     fill: @darkred@
 }
 
+.ZmSuggestView .ZmSuggestHeader div[class*='Img'] .svg-icon {
+	fill:@white@;
+	opacity:0.7;
+}
+
+.ZmSuggestView .ZmSuggestHeader div[class*='Img']:hover .svg-icon {
+    opacity:1;
```
Comment (silentsakky) [inline WebRoot/skins/_base/base4/skin.css]: tabbing looks bit off

Comment (kalpeshhpatel) [reply WebRoot/skins/_base/base4/skin.css]: Corrected: https://github.com/Zimbra/zm-web-client/pull/154/files#diff-6d758c79df985840797b946711380390R925

Thanks

**Pull request merged**